Cat.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Image, Text, View } from 'react-native';
+import kitty from './assets/kitty.jpg'
+
```

Comment:
Nice! You figured out how to load an image from assets. That is a more real world scenario than referencing a URL. The reason why is because unlike a web app, if you lose your internet connection the app is still partially functional. If we loaded all the images from URLs you would not be able to see them if we lost connectivity.